src/Fixie.Tests/MessagingTests.cs

```diff
@@ -37,11 +37,8 @@ protected MessagingTests()
 
         protected string TestClass { get; }
 
-        protected void Discover(Listener listener)
-            => Discover<SampleTestClass>(listener, convention);
-
-        protected void Run(Listener listener)
-            => Run<SampleTestClass>(listener, convention);
+        protected void Run(Listener listener) 
+            => Utility.Run(listener, convention, typeof(SampleTestClass), typeof(EmptyTestClass));
```

Comment:
Because we're "using static Utility" at the top, we can avoid saying "Utility." here. It's basically a global function.

Reply:
If I remove the explicit call, VS or Roslyn tries to interpret it as a recursive call with invalid arguments and fails to compile.